src/ProfileEvent.h

```diff
@@ -308,6 +308,9 @@ class ProfileEventSingleton {
         /** thread timer runs on */
         std::thread th;
 
+        /** number of utilistation events */
+        uint32_t count;
```

Comment:
'count' is a bit generic, given all the things we could be counting. Maybe 'runCount'?

Reply:
Also, initialise it to 0 here, rather than in the constructor